code/game/machinery/Sleeper.dm

```diff
@@ -157,6 +175,40 @@
 /obj/machinery/sleeper/process()
 	..()
 	check_nap_violations()
+	var/mob/living/carbon/C = occupant
+	if(C)
+		if(stasis && C.stat >= stasis_health)
```

Comment:
Should this not be <=

Reply:
stat is weird and the values get higher at worse conditions